src/lib.rs

```diff
@@ -24,6 +24,7 @@ mod lens;
 mod value;
 
 use std::any::Any;
+use std::ops::Deref;
```

Comment:
Generally we would write these on the same line, especially as they're related: `std::ops::{Deref, DerefMut}`